main.js

```diff
@@ -12,4 +12,165 @@ popUp.addEventListener('click', () => {
+
+const openWindow = document.getElementById('modal-box');
+
+function popUpCard(card) {
+  const temp = document.createElement('template');
+  temp.innerHTML = `
+    // <!--pop-up for the mobile portfolio starts-->
+    <div id="mymodal" class="modal">
+      <section class="sectiontemplatepop-up">
+        <div class="Sectionpop-up">
+            <div class="popclose">
+            <p class="h1m">Tonic</p>
+            <span id="closepop">&times;</span>
+            </div> </br>
+            <ul class="list1">
+              <li class="can">Canopy</li>
+              <li class="special">Back End Dev</li>
+              <li class="special">2015</li>
+            </ul>
+            <div class="desktoptech">
+        <img class="imgm1"  src="${card.img1}" alt="Natural Tonic" />
+        <img class="imgdm1" src="${card.imgd1}" alt="Natural Tonic" />
+        <div class="textpop">
+        <p class="pop-uptext">
+        Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry.
+          </p>
+          <p class="pop-uptexti">
+          Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+          </p>
+          <div class=listm>
+          <ul class="list2">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <ul class="listo">
+            <li>github</li>
+            <li>rail</li>
+            <li>bootstrape</li>
+          </ul>
+          </div>
+          <img class="imgproject1" src="./img/Devider.svg" alt="Line" />
+          <div class="modalproject">
+          <div class="project1">
+               <p class="seeprojecti"> See Live</p>  
+                <img class="imgproject" src="./img/Icon.svg" alt="See Live" />
+          </div>
+            <div class="project2">
+               <p class="seeproject"> See Source</p> 
+              <img class="imgprojecti" src="img/Vector.svg" alt="See Source" />
+            </div>
+        </div>
+        </div>
+          </div>
+      </section>
+      <div>
+ `;
+  const box = document.getElementById('modal-box');
+  box.appendChild(temp.content);
+
+  const closeWindow = document.getElementById('closepop');
+  closeWindow.addEventListener('click', () => {
+    openWindow.style.display = 'none';
+  });
```

Comment:
- [ ] Kindly when I click on the `see project` button the modal get appears to me at the first time, but after I close it and try to open it again it doesn't open with me. 
The issue comes from when you close the modal you add a `openWindow.style.display = 'none';` so this `display: none` stays after you close it, so when you try again to open it it doesn't appear because it still has this `display: none'.

You can fix this issue by deleting the modal after you click on the close button instead of adding `display: none`.